package.json

```diff
@@ -1276,6 +1276,7 @@
     "jest-diff": "^29.7.0",
     "joi": "^18.0.1",
     "joi-to-json": "^5.0.0",
+    "joi-to-json-schema": "^5.1.0",
```

Comment:
How different is this one from `joi-to-json` (which is the lib we use for joi->OAS and iirc it does confirm to json schema too?)